apps/mymedicare_cb/tests/test_callback.py

```diff
@@ -56,7 +56,7 @@ def test_callback_url_missing_state(self):
 
     def test_authorize_uuid_dne(self):
         auth_uri = reverse(
-            'dot_ext:authorize-instance',
+            'oauth2_provider:authorize-instance',
```

Comment:
i think this makes sense, it's certainly the smaller change to rename only these calls. However as a developer I wouldn't immediately know where to go to find the behavior this is referencing.

Reply:
LGTM